owners/templates/example.template.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Owners File Syntax</title>
+
+  <style type="text/css">
+    /*
+      github.com style (c) Vasily Polovnyov <vast@whiteants.net>
```

Comment:
What's the license for this file?

Reply:
It's taken directly from `lodash` example files, so it's the same as that repo aka MIT

Reply:
I think that means we can't use it then? I'm not sure how MIT and Apache interact

Reply:
NodeJS is MIT...

Reply:
And MIT license is *more* permissive than Apache

Reply:
Dependencies ≠ inclusion in the codebase...

Reply:
@rsimha are you aware of what the recommendation is for this?

Reply:
Kinda want to resurrect this 🙃